app/forms/candidate_interface/science_gcse_grade_form.rb

```diff
@@ -243,5 +245,11 @@ def reset_missing_explanation!(qualification)
 
       qualification.update(missing_explanation: nil)
     end
+
+    def grade_contains_number?(grade)
+      return false if grade.nil?
+
+      grade.count('0-9').positive?
```

Comment:
Could this just be  ` grade.count('0-9') == 2`

Reply:
Previously I had this as `grade.count('0-9') > 0` but his was corrected by the linter to the above.

I think the intention here is to return true if any number is present rather than explicitly 2. However, thinking about it again it makes sense to just scope this to the double awards. The single awards are currently excluded by default of being part of the ALL_GCSE_GRADES which works but is a bit untidy. 👍